website/templates/emails/reviews_resubmission_confirmation.html.mako

```diff
@@ -0,0 +1,20 @@
+## -*- coding: utf-8 -*-
+<div style="margin: 40px;">
+    <p>Hello ${user.fullname},</p>
+        <p>The ${reviewable.provider.preprint_word} <a href="${reviewable.absolute_url}">${reviewable.node.title}</a> has been successfully re-submitted to ${reviewable.provider.name}.</p>
+        <p>${reviewable.provider.name} has chosen to moderate their submissions using a pre-moderation workflow, which means your submission is pending until accepted by a moderator. You will receive a separate notification informing you of any status changes.</p>
+        <p>You will ${'not receive ' if no_future_emails else 'be automatically subscribed to '}future notification emails for this ${reviewable.provider.preprint_word}. Each ${reviewable.provider.preprint_word} is associated with a project on the Open Science Framework for managing the ${reviewable.provider.preprint_word}. To change your email notification preferences, visit your <a href="${domain + 'settings/notifications/'}">project user settings</a>.</p>
```

Comment:
Use `.format()` instead of `+`. Also, while I'm scrolled so far to the right... we should never ever have lines this long. Could you break it into a few lines?